CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log for Dash for R
 All notable changes to this project will be documented in this file.
 
+## [0.5.0 ] - 2020-05-28
+### Added
+- Dash for R now depends on the `brotli` package explicitly; previously it was loaded when importing `reqres`. [#204](https://github.com/plotly/dashR/pull/204)
+
+### Changed
+- Dash for R no longer wraps the layout in an `htmlDiv` internally, for parity with Dash for Python. Starting in v0.5.0, the `layout` method only accepts a single argument, and that argument must be a Dash component or a function that returns a Dash component. [#121](https://github.com/plotly/dashR/pull/121)
```

Comment:
What does this mean for the majority of apps? Any changes folks will need to do to make sure their apps don't do something odd?

Reply:
@josegonzalez Good question; ultimately, it means that code like this:

```r
app$layout(
  htmlDiv("Some text"),
  htmlDiv("Some more text")
)
```

is no longer treated as valid. It now needs to be wrapped within a div, with the children passed as a list:

```r
app$layout(
  htmlDiv(
    list(
      htmlDiv("Some text"),
      htmlDiv("Some more text")
    )
  )
)
```

This was an artifact of the initial implementation (from the very, very beginning of Dash for R), which supported passing components as `...` (like the R equivalent of `kwargs`). Realistically it's not supported by Dash, but was _possible_ in Dash for R previously.

I'd consider this a breaking change-level modification, but now is the time to :hocho: the offending code, before we reach v1.0.